src/functionHandlers/getProfile/getProfile.ts

```diff
@@ -18,9 +18,22 @@ const getIdentifier = async (event: APIGatewayEvent) => {
   if (!id) {
     throw new createHttpError.BadRequest("Identifier is not provided");
   }
+
+  const apiKey =
+    event.headers["x-api-key"] ||
+    event.headers.Authorization ||
+    event.headers["api-key"] ||
+    event.headers.apikey ||
+    config.sheetsApiKey;
+
+  if (!apiKey) {
+    throw new Error("API key is not provided");
+  }
+
   const identities = await memoize(
     () =>
       sheetsToJson<Identities>({
+        sheetsApiKey: apiKey,
```

Comment:
Dont pass it to the backend! I'ts like what i say, do you want your user to know your MySQL db password?